src/Uno.UI.RuntimeTests/Tests/Windows_UI/Given_Color.cs

```diff
@@ -63,5 +63,25 @@ public static IEnumerable<object[]> GetCompare()
 			yield return new object[] { ColorHelper.FromArgb(0xFF, 0xFF, 0x7F, 0xFF), ColorHelper.FromArgb(0xFF, 0xFF, 0x7F, 0xFF), true };
 			yield return new object[] { ColorHelper.FromArgb(0xFF, 0xFF, 0xFF, 0x7F), ColorHelper.FromArgb(0xFF, 0xFF, 0xFF, 0x7F), true };
 		}
+#if __MACOS__
```

Comment:
Let's enable the feature only on net6 and later for now. You'll need to do the same for the other blocks of this PR.

```suggestion
#if __MACOS__ && NET6_0_OR_GREATER
```